v2/pkg/protocols/websocket/websocket.go

```diff
@@ -86,6 +86,11 @@ type Input struct {
 	Name string `yaml:"name,omitempty" jsonschema:"title=optional name for data read,description=Optional name of the data read to provide matching on"`
 }
 
+const (
+	couldNotParseUrl = "could not parse input url"
+	couldNotEvalTempExp = "could not evaluate template expressions"
```

Comment:
Please do not use abbreviations. When someone sees "Temp" they usually think of "Temporary", not "Template". Also add the `Message` or `ErrorMessage` suffix